test-network/network.sh

```diff
@@ -267,9 +267,22 @@ function networkUp() {
 # call the script to create the channel, join the peers of org1 and org2,
 # and then update the anchor peers for each organization
 function createChannel() {
-  # Bring up the network if it is not already up.
+  # Bring up the network if it is not already up. 
+  bringUpNetwork="false"
 
-  if [ ! -d "organizations/peerOrganizations" ]; then
+  if ! $CONTAINER_CLI info > /dev/null 2>&1 ; then
+    fatalln "$CONTAINER_CLI network is required to be running to create a channel"
+  fi
+
+  # check if all containers are present 
+  CONTAINERS=($($CONTAINER_CLI ps | grep hyperledger/ | awk '{print $2}'))
+  len=$(echo ${#CONTAINERS[@]})
+
+  [ $len -eq 7 ] && [ ! -d "organizations/peerOrganizations" ] && networkDown
+
+  [ $len -lt 7 ] || [ ! -d "organizations/peerOrganizations" ] && bringUpNetwork="true" || echo "Network Running Already"
```

Comment:
when would `bringUpNetwork="true"` be set. I only see it being set one time as false.

Reply:
`bringUpNetwork` should be set to true if either conditional on 283 does not `exit 0`

```
  [ $len -lt 7 ] || [ ! -d "organizations/peerOrganizations" ] && bringUpNetwork="true" || echo "Network Running Already"
```